packages/grid/component.ts

```diff
@@ -1,14 +1,14 @@
 import { StarryUIComponentDefinition } from '@starryui/traits'
 import { grid } from '.'
-import { withTextContent } from '@starryui/traits'
+import { withCellDimensions } from '@starryui/traits'
 import { applyTheme } from '@starryui/theme'
 
-export const buttonDefinition: StarryUIComponentDefinition = {
+export const gridDefinition: StarryUIComponentDefinition = {
  title: 'grid',
  exampleSource: `const themedGrid = applyTheme(theme, grid)
-return themedGrid.add(withTextContent('A grid'))()`,
+return themedGrid.add(withCellDimensions(20, 20))()`,
  example(theme) {
   const themedGrid = applyTheme(theme, grid)
-  return themedGrid.add(withTextContent('A grid'))()
+  return themedGrid.add(withCellDimensions(20, 20))()
```

Comment:
✅ The grid component is created successfully and added to the component list in the documentation